README.md

```diff
@@ -51,21 +141,42 @@ It contains the following information:
 Attention: Keep this output file in a safe, private location.
 If this file is lost, the uploaded file content becomes inaccessible.
 
-## metadata
+### files ingest-upload-metadata
+
+*To be performed by Central Data Stewards.*
+
+Upload all file summary JSONs (produced using the
+[files (batch-)upload](#files-batch-upload) command) from the given directory to the
+running system and make the corresponding files available for download.
+
+This command requires a configuration file as described [here](./ingest_config.md).
+
+### metadata
+
+*To be performed by Central Data Stewards only.*
 
 The metadata label groups metadata related commands.
 
 Some of them require a configuration file as described [here](./metadata_config.md).
 
-# load
+### load
+
+*To be performed by Central Data Stewards only.*
 
 The load command makes files and metadata available to user in the running system.
 
 It needs a configuration parameters as described [here](./load_config.md).
+
 ### generate-catalog-accessions
 
-A command for generating accessions for the metadata catalog. Accessions will be
-stored in a text file.
+A command to generate a token/hash pair for interacting with GHGA Central services.
+
+The generated token file should not be moved to a different system and never be shared
+with another user.
+The token hash (**not the token*) must be shared with the GHGA Central Operation
+Team. This process has to be done only once per data steward and system (if a data
+steward is working with multiple compute environments, one set of credentials per
+environment should be created).
```

Comment:
This seems to be in the wrong subsection (generate-catalog accessions)

Reply:
You are write, I misread. Will fix

Reply:
I competely removed the generate-catalog-accessions from the readme because it does not add to the help that is provided when running --help. Moreover, this is a legacy endpoint that should not be that prominent (You only need it when you already know of it).